docs/content/content/using-index-md.md

```diff
@@ -0,0 +1,113 @@
+
+If this is confusing or you are unfamiliar with Hugo's template hierarchy, visit the various template pages listed below. You may need to find the 'active' template responsible for any particular page on your own site by going through the template hierarchy and matching it to your particular setup/theme you are using. 
+
+- [Home page template](/templates/homepage/)
+- [Content List templates](/templates/list/)
+- [Single Content templates](/templates/content/)
+- [Taxonomy Terms templates](/templates/terms/)
+
+Now that you've got a handle on templates lets recap some Hugo basics to understand how to use an \_index.md file with a List page.
+
+1. Sections and Taxonomies are 'List' pages, NOT single pages.
+2. List pages are rendered using the template heirarchy found in the [Content - List Template](http://localhost:1313/templates/list/) docs.
+3. The Home page is not technically a list page but has [it's own template](/templates/homepage/) at layouts/index.html.
+4. Taxonomy terms pages are 'lists of metadata' not lists of content, so [have their own templates](/templates/terms/). 
+
+Let's put this all together clearly because it's a source of much confusion:
+
+>**\_index.md files used in List pages, Terms pages or the Home page are NOT rendered as single pages or with Single Content templates, they are used to 'inject' frontmatter/content into the respective List pages, Terms pages or the Home page and those pages are rendered using templates listed above.**
+
+Here are a couple of examples to make it clearer...
+
+| \_index.md location               | Page affected             | Rendered by                   |
+| -------------------               | ------------              | -----------                   |
+| /content/post/\_index.md          | site.com/post/            | /layouts/section/post.html    |
+| /content/categories/hugo/index.md | site.com/categories/hugo/ | /layouts/taxonomy/hugo.html   |
+
+## How to display content from \_index.md files
+
+From the information above it should follow that content within an \_index.md file won't be rendered in its own Single Page, it'll be 'injected' into the respective List/Terms/Home page. 
+
+To **_actually display that content_** you need to ensure that the relevant template responsible for rendering the List/Terms/Home page contains (at least) `{{ .Content }}`. 
+
+This is the way to actually display the content within the \_index.md file on the List/Terms/Home page. 
+
+A very simple/naive example of this would be:
+
+```html
+{{ partial "header.html" . }}
+	<main>
+        {{ .Content }}
+		{{ $paginator := .Paginate (where .Site.Pages "Type" "post") }}
+		{{ range $paginator.Pages }}
```

Comment:
To make this less confusing, I would change the above to:

```
{{ range .Paginator.Pages }}
```

Which is syntax that is portable to use in the default list template. The above looks like some home page syntax.